src/components/pages/home/in-progress-collection.tsx

```diff
@@ -15,20 +15,25 @@ const InProgressCollection: FunctionComponent<any> = ({collection}) => {
     series,
     slug,
     collection_progress,
+    progress,
     type,
     path,
     items = [],
   } = collection
 
+  const definedProgress = collection_progress || progress
```

Comment:
disparity between REST and gql apis

Reply:
Could these be normalized in `src/lib/users.ts` and then this component doesn't need to know about API layer concerns?

E.g. before returning `all_progress` from `loadUserProgress`, you could transform it, something like this:

```javascript
// all the graphql fetching stuff ...

const normalizedData = all_progress.data.map(item => {
  const { progress, ...rest } = item;
  return { ...rest, collection_progress: progress }
})
const normalizedAllProgress = { ...all_progress, data: normalizedData };

return normalizedAllProgress;
```

Reply:
they could be normalized in several places above this as another project